variables.tf

```diff
@@ -110,6 +110,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-magento-cloud-patches = {
+      description = "This is a Mage-OS fork of the Magento Cloud Patches project found at https://github.com/magento/magento-cloud-patches."
```

Comment:
Can we use the same format as the other mirror repositories? I would love to have them consistent...

Reply:
I guess here you are referring to the same thing I just asked: do you mean renaming it to `mirror-magento2-cloud-patches`?

Reply:
or maybe here you refer to converting "This is a" to "Mage-OS fork of..." 🤔

Reply:
Here I am referring to the description. I do not care too much about them, but I think they should all have the same format :)